packages/mdjs-core/src/mdjsProcess.js

```diff
@@ -54,6 +55,11 @@ const defaultMetaPlugins = [
  */
 async function mdjsProcess(mdjs, { setupUnifiedPlugins = [] } = {}) {
   const parser = unified();
+  if (!rehypePrism) {
+    rehypePrism = (await import('rehype-prism')).default;
```

Comment:
How I understand to use MJS in CJS files, but definitely would do something better if you have as suggestion.